installscripts/jazz-terraform-unix-noinstances/cognito.tf

```diff
@@ -6,6 +6,9 @@ resource "null_resource" "cognito_user_pool" {
 provisioner "local-exec" {
     command = "${var.cognito_cmd}  ${var.envPrefix} ${var.envPrefix}-api-onboarding ${var.cognito_pool_username} ${var.cognito_pool_password} ${var.jenkinspropsfile}"
   }  
+provisioner "local-exec" {
```

Comment:
This will create multiple userpools. Please update the logic in cognito.sh file, with only a single call to the said file.